docs/advanced/exceptions.rst

```diff
@@ -328,6 +328,30 @@ Alternately, to ignore the error, call `PyErr_Clear
 Any Python error must be thrown or cleared, or Python/pybind11 will be left in
 an invalid state.
 
+Handling warnings from the Python C API
+=====================================
+
+Where possible, use :ref:`pybind11 warnings <warnings>` instead of calling
```

Comment:
This `:ref:` does not work, just like the existing one above (line 307 here).

I don't know if you want to go in fixing the existing problem, up to you.

Here I'd simply write (i.e. replace this entire paragraph, lines 334-336, with):

```
Wrappers for handling Python warnings are implemented in ``pybind11/warnings.h``, which must be ``#include``ed explicitly (in other words, it is not transitively included with ``pybind11/pybind11.h``).
```

Reply:
I have copied your text over ("I don't know if you want to go in fixing the existing problem" -- I would not get into it right now...:)).